src/vl-functional-header.js

```diff
@@ -126,7 +126,20 @@ export class VlFunctionalHeader extends VlElement(HTMLElement) {
     }
 
     _registerBackLink() {
-        this._backLinkElement.addEventListener('click', () => window.history.back());
+        this.backLinkEventListener = () => window.history.back();
+    }
+
+    /**
+     * Zet de click event listener voor de 'Terug' knop. Default: ```() => window.history.back()```
+     *
+     * @param {Function} eventListener - Functie met de uit te voeren handeling als op de terug knop wordt geklikt.
+     */
+    set backLinkEventListener(eventListener) {
```

Comment:
Kunnen we dit dan niet beter exposen als event op de vl-functional header?
Een 'back' event, bijvoorbeeld? En dan kiest de afnemer of hij daar op luistert of niet.

Reply:
Nog andere alternatieven zijn:
- back link elementen exposen zodat er rechtstreeks gewerkt kan worden met het native click event
- volledige link element als slot declareren voor wie wilt overschrijven

Ik vind het wel een beetje vreemd dat we de 'terug' link functionaliteit aanpasbaar willen maken. Ik denk dat het probleem hier niet is dat de vorige link geraadpleegd wordt maar wel dat de vorige link (/login) een foutmelding geeft? Kunnen we de oorzaak van het probleem niet best oplossen? Indien niet, hoe bepalen we wat de terug link is? In dit geval kunnen we kiezen om altijd naar de startpagina te gaan, maar wat doen we bij applicaties die meer dan 2 pagina's hebben?

Reply:
Aangezien in dit geval het aanmelden ertussen zit en wij daar niks in te zeggen hebben, is de oorzaak van het probleem oplossen niet mogelijk lijkt me. Als we echt de terugknop niet willen aanraken en de functionaliteit van de browser back altijd behouden, dan stel ik voor om te werken met de breadcrumb, zie ook: https://overheid.vlaanderen.be/webuniversum/v3/documentation/components/vl-ui-functional-header

Reply:
Besproken en we behouden de implementatie zoals in deze PR eerst werd voorgesteld